html/semantics/interactive-elements/the-dialog-element/inert-svg-hittest.html

```diff
@@ -57,11 +57,11 @@
         reachedTarget = true;
     }, { once: true });
 
+    let wrapperRect = wrapper.getBoundingClientRect();
     await new test_driver.Actions()
-        .pointerMove(0, 0, { origin: wrapper })
+        .pointerMove(wrapperRect.x + 1, wrapperRect.y + 1, { origin: "viewport" })
         .pointerDown()
         .send();
```

Comment:
note to self: this part probably doesn't need fixing (the element _should_ be interactable at this point).